apps/prairielearn/src/question-servers/freeform.ts

```diff
@@ -638,17 +641,21 @@ async function processQuestionServer<T extends ExecutionData>(
     return { courseIssues, data, html: '', fileData: Buffer.from(''), renderedElementNames: [] };
   }
 
-  let result, output;
+  let result: any;
+  let output: string;
   try {
     ({ result, output } = await execPythonServer(codeCaller, phase, data, html, context));
-  } catch (err) {
+  } catch (err: any) {
     const serverFile = path.join(context.question_dir, 'server.py');
     courseIssues.push(
-      new CourseIssueError(`${serverFile}: Error calling ${phase}(): ${err.toString()}`, {
-        data: err.data,
-        fatal: true,
-        cause: err,
-      }),
+      new CourseIssueError(
+        `${serverFile}: Error calling ${phase}(): ${(err as Error).toString()}`,
```

Comment:
Is this necessary? `toString()` on `any` should be fine. Ditto elsewhere here.

Reply:
https://github.com/PrairieLearn/PrairieLearn/pull/12620/commits/5671c6a44b2c579d0bce9c61b26863eea521035f from the conflicting tsconfig.